lib/API/Startup.js

```diff
@@ -59,10 +59,11 @@ module.exports = function(CLI) {
     return hash_map[init_systems[i]];
   }
 
-  CLI.prototype.uninstallStartup = function(platform, cb) {
+  CLI.prototype.uninstallStartup = function(platform, opts, cb) {
     var commands;
     var that = this;
     var actual_platform = detectInitSystem();
+    var service_name = 'pm2-' + (opts.user || process.env.USER);
```

Comment:
This leads to two important matters IMO:

- how can we totally remove every startup files?
- how do we know there isn't another one registered (user might just have forgotten the service)?

I don't know what would be best in term of DX (developer experience) and/or if we need to implement something.